styles/login-detail.scss

```
@@ -0,0 +1,69 @@
+*{
+  font-family: $main-font,sans-serif;
+}
+
+.section{
+  display: flex;
+  justify-content: center;
+  padding-top: 184px;
+}
+
+.main-frame {
+  height: 400px;
+  width: 300px;
+  background-color: $background-grey;
+  border-top: 8px solid $border-login;
+}
+
+.wolox-logo{
+  height: 70px;
+  width: 212.42px;
```

```suggestion
  width: 215px;
```

Decimals doesn't make a big change 😄